config/routes.rb

```diff
@@ -598,6 +598,7 @@
   end
 
   match 'host_wizard' => 'react#index', :via => :get
+  get 'next_architectures' => 'react#index'
```

Comment:
I like reusing the `experimental` namespace
```suggestion
  get 'experimental/architectures' => 'react#index'
```